phabfive/core.py

```diff
@@ -63,16 +61,17 @@ def __init__(self):
 
                 raise PhabfiveConfigException(error)
 
+        url = urlparse(self.conf["PHAB_URL"])
+
+        self.url = f"{url.scheme}://{url.netloc}"
+
         self.phab = Phabricator(
-            host=self.conf.get("PHAB_URL"),
+            host=f"{self.url}/api/",
```

Comment:
This throws away the path. The logic seems a bit off. For this particular "bug" we just want to make sure that:

1. A trailing slash works (it does this today)
2. A missing trailing slash also works (this does not work today)